From e1627a8273125ce7bcdbef750171ffd685ed7c1f Mon Sep 17 00:00:00 2001
From: Xiaoguang Chen <xiaoguang.chen@intel.com>
Date: Thu, 9 Jul 2015 21:47:02 +0800
Subject: [PATCH 115/403] kvmgt porting configuration update

Add an entry in the Kconfig file so user can select/deselect the
kvmgt feature using make menuconfig.
Enable KVMGT related features in config-3.18.0-dom0.

Signed-off-by: Jike Song <jike.song@intel.com>
Signed-off-by: Xiaoguang Chen <xiaoguang.chen@intel.com>
diff --git a/config-3.18.0-dom0 b/config-3.18.0-dom0
index ba40749..7060772 100644
--- a/config-3.18.0-dom0
+++ b/config-3.18.0-dom0
@@ -4033,8 +4033,20 @@ CONFIG_CRYPTO_HW=y
 # CONFIG_CRYPTO_DEV_QAT_DH895xCC is not set
 # CONFIG_ASYMMETRIC_KEY_TYPE is not set
 CONFIG_HAVE_KVM=y
+CONFIG_HAVE_KVM_IRQFD=y
+CONFIG_HAVE_KVM_IRQ_ROUTING=y
+CONFIG_HAVE_KVM_EVENTFD=y
+CONFIG_KVM_APIC_ARCHITECTURE=y
+CONFIG_KVM_MMIO=y
+CONFIG_KVM_ASYNC_PF=y
+CONFIG_HAVE_KVM_MSI=y
+CONFIG_HAVE_KVM_CPU_RELAX_INTERCEPT=y
+CONFIG_KVM_VFIO=y
 CONFIG_VIRTUALIZATION=y
-# CONFIG_KVM is not set
+CONFIG_KVM=m
+CONFIG_KVM_INTEL=m
+CONFIG_KVM_DEVICE_ASSIGNMENT=y
+CONFIG_KVMGT=y
 CONFIG_BINARY_PRINTF=y
 
 #
